.github/kokoro/db-full.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
+# SPDX-License-Identifier: ISC
```

Comment:
The SPDX codes go after the license, see the other repository.

Reply:
I thought I did exactly as I saw in the other repo...
Not the point though. Is such header the desired one?
<pre>
#!/bin/bash
#
# Copyright (C) 2017-2020  The Project X-Ray Authors.
#
# Use of this source code is governed by a ISC-style
# license that can be found in the LICENSE file or at
# https://opensource.org/licenses/ISC
#
# SPDX-License-Identifier: ISC
#
</pre>

Reply:
@mithro if the header I pasted above is correct, just let me know and will go on with updating them accordingly.

Reply:
Extra `#` at the end here.